Guidelines for scientific documentation (#77)
CABLE-LSM/CABLE: documentation/docs/developer_guide/doc_guide/index.md

# Documentation guidelines

> The documentation for CABLE is made of two different entities:

Reviewer: ```suggestion
The documentation for CABLE has two parts:
```

>  - the API documentation is based on inline documentation in the source code. This documentation includes the detailed scientific documentation. It is written using [FORD][ford].

Reviewer: ```suggestion
 - inline documentation in the source code. This documentation includes the detailed scientific documentation. It is written using [FORD][ford].
```

>  - the standalone guides such as the User guide and the Developer guide. It is written using [Material for MkDocs][material].

Reviewer: ```suggestion
 - the standalone guides such as the User Guide and the Developer Guide. It is written using [Material for MkDocs][material].
```

All documentation is located in the [CABLE GitHub repository][cable-repo]. Both tools use [Markdown][md-cheatsheet] and [LaTeX Mathematics][latex-maths] for formatting. You can find a cheatsheet for FORD [on this page][ford-cheatsheet].

> To update the documentation, you need to follow a traditional GitHub contribution process with opening an issue and a branch, working on your changes and submitting a pull request for review before publication. The process is explained in more details [on this page][git-process].

Reviewer: ```suggestion
To update the documentation, you need to follow a traditional GitHub contribution process: open an issue and a new branch; work on your changes; submit a pull request for review before publication. The process is explained in more details [on this page][git-process].
```
